modules/modules.js

```diff
@@ -0,0 +1,84 @@
+export default class Books {
+  constructor() {
+    this.booksArr = [];
+    this.name = document.getElementById('name');
+    this.author = document.getElementById('author');
+    this.add = document.getElementById('form');
+    this.container = document.querySelector('.added-books');
+    this.timeContainer = document.querySelector('.localTime');
+    this.updateTime = document.addEventListener('DOMContentLoaded', this.updateTime.bind(this));
+    this.displayBooks();
+    this.add.addEventListener('submit', (e) => {
+      e.preventDefault();
+      this.addBook();
+    });
+
+    this.container.addEventListener('click', (e) => {
+      if (e.target.classList.contains('remove')) {
+        const index = e.target.dataset.id;
+        this.removeBook(index);
+      }
+    });
+  }
+
+  updateTime() {
+    setInterval(() => {
+      const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+      const suffixes = ['st', 'nd', 'rd', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'st', 'nd', 'rd', 'th', 'th', 'th', 'th', 'th', 'th', 'th', 'st'];
+
+      const date = new Date();
+      const month = months[date.getMonth()];
+      const day = date.getDate();
+      const suffix = suffixes[day - 1];
+      const year = date.getFullYear();
+      const hour = date.getHours() % 12 || 12;
+      const minute = date.getMinutes();
+      const second = date.getSeconds();
+      const ampm = date.getHours() >= 12 ? 'pm' : 'am';
+
+      const formattedDate = `${month} ${day}${suffix} ${year}, ${hour}:${minute}:${second} ${ampm}`;
+
+      this.timeContainer.innerHTML = formattedDate;
+    }, 1000);
+  }
+
```

Comment:
- [ ] You did a good job adding the date. However, I would suggest you instead of doing all this work let's ensure to use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library to make adding the date easier. And because it's a requirement I would advise you to stick to it. :ok_hand: 😊

#### Requirement :point_down: 
![Screenshot from 2023-05-01 20-41-54](https://user-images.githubusercontent.com/89228743/235527625-075acaaf-ab19-4d59-9fc4-76f88abcc86b.png)

Reply:
requirement is to display de current date below nav bar, thats already done, its recomends to use luxon but using it is not the requirement.... @ridwanediallo

Reply:
@gersonhiggins yes that's true, the requirement said to display the current date below the nav bar but the same time it says `For this requirement you should use a Library Luxon`.

Reply:
#### This :point_down:  is a screenshot of the requirement 
![Screenshot from 2023-05-01 20-41-54](https://user-images.githubusercontent.com/89228743/235527625-075acaaf-ab19-4d59-9fc4-76f88abcc86b.png)

Reply:
exactly. it says "Display the current date below the navigation bar"
for this requirement - minds current date below bar - you "shloud use a library luxon, luxon is not the requirement, the requirement is the current date, it is already displayed on project
@ridwanediallo

Reply:
@gersonhiggins please if you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._